cmd/backup/main.go

```diff
@@ -273,6 +328,19 @@ func newScript() (*script, error) {
 		})
 	}
 
+	tmpl := template.New("")
+	tmpl.Funcs(template.FuncMap{
+		"formatTime": func(t time.Time) string {
```

Comment:
Is this currently used at all?

Reply:
What is? Are you referring to the `template.New("")` call?

Reply:
Sorry for the confusion. I was referring to the `formatTime` function that does not seem to be used in the templates right now.

Reply:
I was 100% sure I added it in the default template. I probably forgot, but it is needed because of how instances of `Time` are converted to string by default (see here https://go.dev/play/p/s10JwKYEVzX)

In the upcoming changes it will surely be used in the default template.

Reply:
Ok, brilliant.